widgets/accordion/js/accordion.js

```diff
@@ -156,7 +156,11 @@ jQuery( function ( $ ) {
 					updateHash();
 				}
 				var initialScrollPanel = $widget.data( 'initialScrollPanel' );
-				if ( initialScrollPanel > 0 ) {
+				if ( window.location.hash && openPanels.length ) {
+					setTimeout( function () {
```

Comment:
Thanks, Alex. As far as I'm aware `function (` should be `function(`
https://make.wordpress.org/core/handbook/best-practices/coding-standards/javascript/#indentation-and-line-breaks

Reply:
Fixed.